src/engine/map/MapFunctions.hxx

```diff
@@ -197,4 +204,4 @@ private:
   std::vector<uint8_t> calculateAutotileBitmask(Point coordinate);
 };
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
What is this `#endif` for? I don't see any preprocessor if statements in this file.

Reply:
Ah, alright. Admittedly I looked at this after a day of work that fried my brain, so I must have misread the diff.